packages/valaxy/client/main.ts

```diff
@@ -36,6 +39,9 @@ routes.forEach((i) => {
   })
 })
 
+if (useVueRouter)
```

Comment:
Just a question, can we lift `initValaxyConfig` and use config directly to create routes? (instead of virtual module `/@valaxyjs/redirects`).

Reply:
Ok, I will modify it.